__tests__/app.test.js

```diff
@@ -134,4 +134,37 @@ describe("app", () => {
         });
     });
   });
+  describe("GET /api/reviews/:review_id", () => {
+    it("200: GET - This test should respond with the data for the specific review that has been requested in the path (in this case review 1)", () => {
+      return request(app)
+        .get("/api/reviews/1")
+        .expect(200)
+        .then(({ body }) => {
+          const { reviews } = body;
+          expect(reviews).toEqual({
```

Comment:
A better matcher here would be toMatchObject or objectContaining to allow us to expand this test later :)